localstack/entrypoint/init-aws.sh

```diff
@@ -13,56 +13,3 @@ for email in "${VALID_EMAILS[@]}"; do
 done
 
 awslocal s3api create-bucket --bucket cpf-reporter --region us-west-2 --create-bucket-configuration '{"LocationConstraint": "us-west-2"}'
-
-awslocal lambda create-function \
-  --function-name localstack-js-lambda-example \
-  --runtime nodejs16.x \
-  --zip-file fileb:///tmp/function.zip \
-  --handler index.handler \
-  --role arn:aws:iam::000000000000:role/lambda-role
-
-awslocal lambda create-function \
-  --function-name localstack-py-lambda-example \
-  --runtime python3.9 \
-  --zip-file fileb:///tmp/function_py.zip \
-  --handler sample_lambda.lambda_handler \
-  --role arn:aws:iam::000000000000:role/lambda-role
-
-#
-# The js lambdas can be a bit tricky for inputs,
-# so instead we use two python lambdas strung together
-#
-awslocal stepfunctions create-state-machine \
-  --name "run-sample-functions" \
-  --definition '{
-    "Comment": "Invoke Two Lambda Functions",
-    "StartAt": "Lambda1",
-    "States": {
-        "Lambda1": {
-            "Type": "Task",
-            "Resource": "arn:aws:states:::lambda:invoke",
-            "Parameters": {
-                "FunctionName": "localstack-py-lambda-example",
-                "Payload": {
-                    "input.$": "$"
-                }
-            },
-            "OutputPath": "$.Payload",
-            "Next": "InvokeLambda"
-        },
-        "InvokeLambda": {
-            "Type": "Task",
-            "Resource": "arn:aws:states:::lambda:invoke",
-            "Parameters": {
-                "FunctionName": "localstack-py-lambda-example",
-                "Payload": {
-                    "input.$": "$"
-                }
-            },
-            "OutputPath": "$.Payload",
-            "End": true
-        }
-    }
-  }' \
-  --role-arn "arn:aws:iam::000000000000:role/stepfunctions-role"
-
```

Comment:
@tzinckgraf this is the deletion I was referring to. This particular script was helpful for me to test things in local stack but if you think we can do this differently I'm good with removing it.

Reply:
I ended up moving most of that to the script `python/tests/test-create-archive.sh`. I found it was easier to rerun and tweak in that script, and it was a bit hidden in the init-aws script. I can move it back if you prefer.

Reply:
ah gotcha no worries! I missed this update makes sense 👍